.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
   - eval `opam config env`
   - psql -c 'create database links;' -U postgres
 install:
-  - opam install -y deriving lwt postgresql sqlite3 mysql
+  - opam install -y deriving lwt postgresql sqlite3 mysql conduit cohttp
```

Comment:
Stefan, is `conduit` necessary here? It _seems_ to me that this is a transitive dependency, ie. `conduit` is required by `cohttp`, but is not directly used by Links. @jgbm, is that correct? If so than just installing `cohttp` will be fine.